### PR TITLE
Refractor Bayes factors

### PR DESCRIPTION
**Pull request by mattansb**
Return log(BF)

- [x] bf_models
    - [x] as.matrix
    - [x] update
    - [x] format/print
- [x] bf_inclusion
    - [x] format/print
- [x] bf_params
    - [x] format/print
- [x] bf_restricted
    - [x] format/print

### CODE REVIEW

**Comment by bwiernik** [comment]
I think this is a good approach. Printing could show both logBF and BF? Not sure we need a special function to return just the BF considering a user could just do exp(logBF) from the object

File: NEWS.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # bayestestR 0.9.0.00001
 
+## Breaking
+
+- All Bayes factors are now returned as `log(BF)` (column name `log_BF`). Printing is unaffected.
```
Comment (DominiqueMakowski) [inline NEWS.md:5]:
```suggestion
- All Bayes factors are now returned as `log(BF)` (column name `log_BF`). Printing is unaffected. To retrieve the raw BFs, you can run `exp(result$log_BF)`.
```

copypasta examples ftw 😁

**Comment by mattansb** [comment]
@bwiernik You mean to print both side by side be default?

**Comment by bwiernik** [comment]
I don't know, maybe? What does that look like?

**Comment by mattansb** [comment]
As far as I can tell, everything is working (failed tests have nothing to do with my changes).

@DominiqueMakowski Do you think we should add this info in the start up message?

**Comment by strengejacke** [comment]
How does this affect interpretation rules?

**Comment by DominiqueMakowski** [comment]
> Do you think we should add this info in the start up message?

I would tend to say no, since the change is made explicit in the column name and the printing remains correct

**Comment by mattansb** [comment]
> How does this affect interpretation rules?

Just pass `exp(log_BF)`:

``` r
library(bayestestR)

lm1 <- lm(Sepal.Length ~ 1, data = iris)
lm2 <- lm(Sepal.Length ~ Species, data = iris)
lm3 <- lm(Sepal.Length ~ Species + Petal.Length, data = iris)
lm4 <- lm(Sepal.Length ~ Species * Petal.Length, data = iris)

BFM <- bayesfactor_models(lm1, lm2, lm3, lm4, denominator = lm1)

effectsize::interpret_bf(exp(BFM$log_BF), include_value = T)
#> [1] "no evidence (BF = 1.00) against or in favour of"
#> [2] "extreme evidence (BF = 1.70e+29) in favour of"  
#> [3] "extreme evidence (BF = 5.84e+55) in favour of"  
#> [4] "extreme evidence (BF = 2.20e+54) in favour of"  
#> (Rules: jeffreys1961)
```

<sup>Created on 2021-05-12 by the [reprex package](https://reprex.tidyverse.org) (v1.0.0)</sup>

**Comment by strengejacke** [comment]
Ok, so we _save_ log(BF) in the df, but we _print_ BF, right? Maybe we could add an `as.bayesfactor()` or `as.bf()` method, that just makes `exp(log_BF)`, so users can be sure to have the "correct" BF. I'm not sure why we did this change - just because it fails in certain instances? If we never print log(BF) anyway, we could just return `Inf` and give a warning, saying that some BF are beyond what humans can imagine (and machines can precisely calculate).

**Comment by DominiqueMakowski** [comment]
Also, if it's a problem of memory precision, knowing that the logarithm also tends to infinity there might be such a big BF that even its log is outside the machine precision no? <sub>(it's the magnitude of the BF that 0.95 > 0.89 😬)</sub>

**Comment by mattansb** [comment]
This was does because (1) the lack of precision, and (2) caused issues in `bf_inclusion` even when those resulted in non-astronomical BFs (see #420).

> If we never print log(BF) anyway...

It is an option in the printing functions.

**Comment by mattansb** [comment]
I'm done here. @DominiqueMakowski I think you can merge.

**Comment by DominiqueMakowski** [comment]
does @strengejacke put his veto?

**Comment by strengejacke** [comment]
Go!

**Pull request merged**